packages/myst-cli/src/transforms/images.ts

```diff
@@ -220,13 +221,24 @@ export async function transformImageFormats(
       session.log.info(`${logPrefix} PNG using inkscape`);
       svgConversionFn = inkscape.convertSvgToPng;
     } else if (svgToPng && imagemagickAvailable) {
+      if (svgToPdf) {
+        addWarningForFile(
```

Comment:
Nice, this is good. I think there's an edge case where only `pdf` is in the allowed image formats and not `svg` or `png` - so in that specific case we are converting from one unsupported format to another (when we might just prefer to leave the original unsupported `svg`).

Reply:
Hm yeah, good point. I am good to kick that down the road, as it is a pretty unlikely scenario.